build.gradle

```diff
@@ -25,18 +25,20 @@ subprojects {
 
     def junit5Version = '5.6.0'
     def jacksonVersion = '2.10.2'
+    def awsSdkVersion = '1.11.714'
 
     dependencies {
-        implementation group: 'com.github.BIBSYSDEV', name: 'nva-commons', version: 'v0.1.5'
+        implementation group: 'com.github.BIBSYSDEV', name: 'nva-commons', version: '07fdf93'
```

Comment:
We need to release a new version nva-commons before this is merged

Reply:
v.0.1.6